src/connection/server_support.c

```diff
@@ -1848,16 +1821,12 @@ css_init (THREAD_ENTRY * thread_p, char *server_name, int name_length, int port_
       er_set (ER_FATAL_ERROR_SEVERITY, ARG_FILE_LINE, ER_GENERIC_ERROR, 0);
       return ER_FAILED;
     }
-
-  /* startup worker/daemon threads */
-  status = thread_start_workers ();
-  if (status != NO_ERROR)
+  css_Connection_worker_pool = cubthread::get_manager ()->create_worker_pool (MAX_WORKERS, MAX_TASK_COUNT, NULL,
```

Comment:
Why MAX_TASK_COUNT is twice number of transactions ?
I think this is the relic from old code, do we still need it ?

Reply:
Actually the parameter doesn't have any effect. It was previously used to determine the size of lock-free circular queue (task queue), but now it only impacts try_execute.

Now that you asked, I realize that connections should never queue a task. try_execute should work immediately. I will change the number of tasks to be same as the number of threads and use try_execute instead of execute. If try_execute fails, it should be a design issue.